### ISSUE TITLE
Cannot re-use `SlackMessageContent` from Events /w Web API

### ISSUE THREAD

**Issue by medwards** [abdolence/slack-morphism-rust#24]
If I am writing an echo bot (ie sends identical messages back when receiving messages) then I cannot re-use the `SlackMessageContent` struct I receive from the Events API, ie
```rust
                content: SlackMessageContent {
                    text: Some(
                        "p",
                    ),
                    blocks: Some(
                        [
                            RichText,
                        ],
                    ),
                },
```

results in
```
    ApiError(
        SlackClientApiError {
            code: "invalid_blocks",
            warnings: None,
            http_response_body: Some(
                "{\"ok\":false,\"error\":\"invalid_blocks\",\"errors\":[\"missing required field: elements [json-pointer:\\/blocks\\/0]\"],\"response_metadata\":{\"messages\":[\"[ERROR] missing required field: elements [json-pointer:\\/blocks\\/0]\"]}}",
            ),
        },
    ),
```

Snippet of sample code (its an extension of the example)
```rust
...
                    listener.push_events_service_fn(
                        thread_push_events_config,
                        test_push_events_function,
                    ),
...
async fn test_push_events_function(event: SlackPushEvent, _client: Arc<SlackHyperClient>) {
    println!("{:#?}", event);
    if let SlackPushEvent::EventCallback(callback) = event {
        if let SlackEventCallbackBody::Message(message) = callback.event {
            if let Some(channel_type) = message.origin.channel_type {
                if channel_type == SlackChannelType("im".to_owned()) {
                    let resp = send_message(
                        &message
                            .origin
                            .channel
                            .expect("Message didn't have a channel"),
                        &message.content,
                    )
                    .await; // TODO catch failures here and log them
                }
            }
        }
    }
}


async fn send_message(
    channel: &SlackChannelId,
    content: &SlackMessageContent,
) -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
    println!("triggered");
    let request = SlackApiChatPostMessageRequest::new(channel.clone(), content.clone());

    let hyper_connector = SlackClientHyperConnector::new();
    let client = SlackClient::new(hyper_connector);
    let token_value: SlackApiTokenValue = config_env_var("SLACK_BOT_TOKEN")?.into();
    let token: SlackApiToken = SlackApiToken::new(token_value);
    let session = client.open_session(&token);

    let response = session.chat_post_message(&request).await?;
    println!("chat_post_message response: {:#?}", response);
    Ok(())
}
```

**Comment by medwards** [comment]
For a complete example see: https://github.com/medwards/slack_echo_bot/blob/077c1e8d97f03882a38218af8f4ce3a9374e301d/src/main.rs#L44-L55

I was expecting to be able to just pass `message.content.clone()`

**Comment by abdolence** [comment]
Hm, this is interesting, let me check. Thanks for the report!

**Comment by abdolence** [comment]
Ok, I probaly know the reason.
This is about support for `RichText` blocks, and I think I can know how to fix it. I'll give a try this weekend.

**Comment by abdolence** [comment]
Should be fixed now in v0.6.0.

**Comment by abdolence** [comment]
By the way:
> send_message 
here you can reuse one client from events. Creating a hyper connector might be expensive, so I recommend use one client everywhere.
